ecommerce/migrations/0022_backfill_reference_number.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.14 on 2022-08-09 16:25
+
+from django.db import migrations
+from ecommerce.constants import REFERENCE_NUMBER_PREFIX
+from django.conf import settings
+
+
+def backfill_order_reference_number(apps, schema_editor):
+    Order = apps.get_model("ecommerce", "Order")
+
+    for order in Order.objects.all():
+        order.reference_number = (
+            f"{REFERENCE_NUMBER_PREFIX}{settings.ENVIRONMENT}-{order.id}"
+        )
+        order.save()
```

Comment:
Might be faster to run this as an `UPDATE` query. See https://github.com/mitodl/micromasters/pull/5216/files#diff-604f0a44454c778f479a44b46400eda6f693a4963ded753a4000750ab0d67b78R15-R22